src/fidesops/ops/service/strategy.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import logging
+from abc import ABC
+from typing import Any, Dict, Generic, List, Optional, Type, TypeVar
+
+from pydantic import ValidationError
+
+from fidesops.ops.common_exceptions import NoSuchStrategyException
+from fidesops.ops.common_exceptions import ValidationError as FidesopsValidationError
+from fidesops.ops.schemas.saas.strategy_configuration import StrategyConfiguration
+
+logger = logging.getLogger(__name__)
+T = TypeVar("T", bound="Strategy")
+C = TypeVar("C", bound=StrategyConfiguration)
```

Comment:
i'm a bit unsure on this type var. this was the simplest way i could find to allow subclasses to define their own subtype of `StrategyConfiguration` for their `configuration_model` class var, without getting mypy failures. but i may be overlooking something more straightforward and/or idiomatic.